src/main/java/tornadofx/Drawer.kt

```diff
@@ -23,7 +24,7 @@ fun EventTarget.drawer(
         op: Drawer.() -> Unit
 ) = Drawer(side, multiselect, floatingContent).attachTo(this, op)
 
-class Drawer(side: Side, multiselect: Boolean, floatingContent: Boolean) : BorderPane() {
+class Drawer(side: Side, multiselect: Boolean, floatingContent: Boolean, horizontalItem: Boolean) : BorderPane() {
```

Comment:
it might be worth renaming. For example, by adding the prefix ʻis`, it is possible to remove ʻItem`.